packages/vuepress-theme-odyssey/styles/templates/_OdoTemplateHome.scss

```diff
@@ -0,0 +1,45 @@
+.odo-hero {
+  margin-bottom: $spacing-l;
+  padding: $spacing-m;
+  border-radius: 0 32px;
+  background-color: var(--odo-card-bg);
+
+  &.odo-hero--layout-l {
+    display: grid;
+    grid-column-gap: $spacing-l $spacing-l;
+    grid-template-columns: repeat(12, 1fr);
```

Comment:
Do we need there to be an even 12-column split? Could we use content-based measurements here?

If there's something that needs to be a certain size, we can also account for that and leave the rest fluid.